waspc/data/Generator/templates/react-app/vite.config.ts

```diff
@@ -15,7 +15,7 @@ const defaultViteConfig = {
   base: "{= baseDir =}",
   plugins: [react()],
   server: {
-    port: 3000,
+    port: {= defaultClientPort =},
```

Comment:
Interesting, why did we extract this?

Reply:
Answered here: https://github.com/wasp-lang/wasp/pull/1516#discussion_r1371466384